packages/editor-ui/src/components/ParameterInputList.vue

```diff
@@ -23,11 +23,7 @@
 			</div>
 
 			<import-parameter
-				v-else-if="
-					parameter.type === 'curlImport' &&
-					nodeTypeName === 'n8n-nodes-base.httpRequest' &&
-					nodeTypeVersion >= 3
```

Comment:
would not this now show for old versions of http node? I imagine we need something like
`parameter.type === 'curlImport' && ((nodeTypeName === 'n8n-nodes-base.httpRequest' &&
					nodeTypeVersion >= 3) || isHTTPRequestedBase(nodeType))`

Reply:
Only the HTTP Request V3 Node has a parameter with type == `curlImport`, so old versions don't show it. I thought these checks were not needed, as no other node has `curlImport` as a parameter type. Let me know if I'm missing something, I'll revert it

Reply:
Perfect. much better this way